src/test/java/teammates/test/cases/action/InstructorFeedbackEditCopyActionTest.java

```diff
@@ -268,16 +268,16 @@ public void testExecuteAndPostProcess() {
 
         assertEquals("", editCopyData.redirectUrl);
 
-        expectedString = "\"\" is not acceptable to TEAMMATES as a/an feedback session name because it is empty. "
+        expectedString = "The field feedback session name is empty. "
```

Comment:
@damithc It won't take more than a commit if you say I can do that in this PR only.